package.json

```diff
@@ -108,11 +113,11 @@
     "stylelint-scss": "3.9.2",
     "superagent": "5.1.0",
     "ts-node": "8.2.0",
-    "typescript": "3.7.5"
+    "typescript": "3.9.5"
   },
   "engines": {
     "node": ">=12 <13",
-    "yarn": ">=1.22.4 <2",
+    "yarn": ">=1.17.0 <2",
```

Comment:
More of a question why we are lowering the minimum version number?

Reply:
It looks like I missed this change when I rebased. I'll update to what is on master branch.